Tests/KituraTests/TestTypeSafeMiddleware.swift

```diff
@@ -48,7 +48,7 @@ class TestTypeSafeMiddleware: KituraTest {
             ("testSingleMiddlewarePut", testSingleMiddlewarePut),
             ("testMultipleMiddlewarePut", testMultipleMiddlewarePut),
             ("testSingleMiddlewarePatch", testSingleMiddlewarePatch),
-            ("testMultipleMiddlewarePatch", testMultipleMiddlewarePatch),
```

Comment:
Is there a specific "disallow commas at the end of lists" rule? If so, I'd consider disabling it rather than making this change.

Specifically, I like the ability to include the same delimiters on all lines of the list (particularly when it is laid out with the opening and closing braces on separate lines, as is the case here).  It means that adding new entries is a simpler operation, and if adding to the end of the list, produces a cleaner diff.